### PR TITLE
Add support for reachable status

### PR DESCRIPTION
**Pull request by schmittx**
Add support for `reachable` status.

This isn't described in the Netatmo Dev docs, but I confirmed it's in the response for all of my weather station modules. I don't have HomeCoach so I can't confirm that, which is why I've excluded it for now. If someone wants to check that and modify this PR, please do so!

image

### CODE REVIEW

File: smart_home/WeatherStation.py [hunk 1]
```diff
@@ -106,6 +106,7 @@ def moduleById(self, mid, sid=None):
     def monitoredConditions(self, module):
         mod = self.moduleByName(module)
         conditions = []
+        conditions.extend("reachable")
```
Comment (jabesq) [inline smart_home/WeatherStation.py]: This line will return conditions =[r,e,a,...]. You should do conditions.extend([”reachable”]) or conditions.append(”reachable”)

**Review by jabesq — CHANGES_REQUESTED** [review]

File: smart_home/WeatherStation.py (hunk 1 above)
Comment (schmittx) [reply smart_home/WeatherStation.py]: Yeah, good catch. Although this specific line should actually be deleted. I missed this when excluding support for HomeCoach. This same error was fixed lower down though.

**Review by jabesq — APPROVED** [review]

**Comment by schmittx** [comment]
@jabesq Thanks for merging this, can you publish an to update to PyPi now?

**Comment by jabesq** [comment]
I just released it with pyatmo 2.3.0

**Pull request merged**